src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1368,19 +1368,33 @@ namespace
                 }
             }
 
+            asf::Vector3d UVFlip_vec(-1,1,1);
```

Comment:
We use snake_case for variable names: `uv_flip_vec`.

Add spaces after commas.

Spell out floating constants: 1.0.